intercom_2n/utils.py

```diff
@@ -0,0 +1,33 @@
+import os
+import socket
+import struct
+import fcntl
+
+def get_lan_ip():
+    try:
+        ip = socket.gethostbyname(socket.gethostname())
+        if ip.startswith("127.") and os.name != "nt":
+            interfaces = ["eth0", "eth1", "eth2", "wlan0", "wlan1", "wifi0", "ath0", "ath1", "ppp0"]
+            for ifname in interfaces:
+                try:
+                    ip = get_interface_ip(ifname)
+                    break
+                except IOError:
+                    pass
+    except socket.gaierror:
+        return get_lan_ip_fallback()
+    return ip
+
+def get_interface_ip(ifname):
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    return socket.inet_ntoa(fcntl.ioctl(s.fileno(), 0x8915, struct.pack('256s', ifname[:15].encode('utf-8')))[20:24])
```

Comment:
The problem is gone :)
I can just delete the utils.py. I've used it because i had first implemented a http-listener. This is now removed because i using the more stable pulli mode.